profile/2018-03-01-hybrid.json

```diff
@@ -0,0 +1,124 @@
+{
+    "info":{
+        "name":"2018-03-01-profile",
+        "description":"Profile definition targeted for hybrid applications that could run on azure stack 1807 version and azure cloud"
+    },
+    "resource-manager": {
+        "microsoft.authorization": {
+			"2017-04-01": [
+				"locks"
+			],
+			"2015-07-01": [
+				"operations",
+				"permissions",
+				"roleAssignments",
+				"roleDefinitions"
+			],
+			"2016-12-01": [
+				"policyAssignments",
+				"policyDefinitions"
+			],
+			"2015-07-01-preview": [
+				"providerOperations"
+			]
+        },
+        "microsoft.compute": {
+			"2017-03-30": [
+				"availabilitySets",
+				"locations",
+				"disks",
+				"diskoperations",
+				"locations/diskoperations",
+				"locations/publishers",
+				"locations/operations",
+				"locations/usages",
+				"locations/vmSizes",
+				"operations",
+				"snapshots",
+				"virtualMachines",
+				"virtualMachines/extensions",
+				"virtualMachineScaleSets",
+				"virtualMachineScaleSets/extensions",
+				"virtualMachineScaleSets/networkInterfaces",
+				"virtualMachineScaleSets/virtualMachines",
+				"virtualMachineScaleSets/virtualMachines/networkInterfaces"
+			]
+        },
+        "microsoft.keyvault": {
+			"2016-10-01": [
+				"operations",
+				"vaults",
+				"vaults/accessPolicies",
+				"vaults/secrets"
+			]
+        },
+        "microsoft.network": {
+			"2017-10-01": [
+				"connections",
+				"loadBalancers",
+				"localNetworkGateways",
+				"locations",
+				"locations/operationResults",
+				"locations/operations",
+				"locations/usages",
+				"networkInterfaces",
+				"networkSecurityGroups",
+				"operations",
+				"publicIpAddresses",
+				"routeTables",
+				"virtualNetworkGateways",
+				"virtualNetworks"
+			],
+			"2016-04-01": [
+				"dnsZones"
+			]
+        },
+        "microsoft.resources": {
+			"2017-10-01": [
+				"deployments",
+				"deployments/operations",
+				"links",
+				"locations",
+				"operations",
+				"providers",
+				"resourceGroups",
+				"resources",
+				"subscriptions",
+				"subscriptions/locations",
+				"subscriptions/operationresults",
+				"subscriptions/providers",
+				"subscriptions/resourceGroups",
+				"subscriptions/resourceGroups/resources",
+				"subscriptions/resources",
+				"subscriptions/tagNames",
+				"subscriptions/tagNames/tagValues",
+				"tenants"
+			],
+			"2015-01-01": [
```

Comment:
version doesn't exist

Reply:
decided in f2f meeting that we should remove this version completely since types are only for Stack